languages/translations/pt-BR.json

```diff
@@ -132,32 +132,27 @@
             "sharp left": {
                 "default": "Faça uma curva fechada a esquerda",
                 "name": "Faça uma curva fechada a esquerda para manter-se na {way_name}",
-                "destination": "Faça uma curva fechada a esquerda sentido {destination}",
-                "junction_name": "Faça uma curva fechada a esquerda em {junction_name}"
```

Comment:
Did you mean to remove the junction_name phrases?

Reply:
Oof, that’s a bad merge that crept in when rebasing.

Reply:
Or rather, it probably got overridden because I ran `npm run transifex` before Transifex had synchronized with the changes in #291.